This patch is part of a series of patches for festival for
Debian GNU/Linux, to provide ncurses commandline editing for Debian

Index: b/config/modules/editline.mak
===================================================================
--- a/config/modules/editline.mak
+++ b/config/modules/editline.mak
@@ -43,10 +43,10 @@
 MOD_DESC_EDITLINE=Use editline for command line editing and history
 
 IO_DEFINES   += -DSUPPORT_EDITLINE $(MODULE_EDITLINE_OPTIONS:%=-DEDITLINE_%)
-MODULE_LIBS  += $(TERMCAPLIB)
+MODULE_LIBS  += -lncurses
+#MODULE_LIBS  += $(TERMCAPLIB)
 
 ifeq ($(DIRNAME),siod)
     CSRCS := $(CSRCS) el_complete.c editline.c el_sys_unix.c
 endif
 
-
